receiver/syslogreceiver/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/syslog
 go 1.19
 
 require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.0.0-00010101000000-000000000000
```

Comment:
Should we fudge the version number here or wait til next release for it to be captured there?

Reply:
You're right. `coreinternal` is not a module created in this PR. I updated it